components/FollowButton.vue

```diff
@@ -17,13 +17,22 @@
 <script setup lang="ts">
 const props = defineProps<{ address: string, owner: string }>()
 const abc = useArtByCity()
-const loading = ref(false)
 
 const {
   data: following,
   pending,
   refresh
 } = useLazyAsyncData(`follows-for-${props.owner}`, async () => {
+  const contract = await abc.connect().following.getContract(props.owner)
```

Comment:
This should not be done every time we load it.  We definitely should not ever *write* things triggered from a page load if it has nothing to do with analytics.  Especially not deploying a contract.  As you have it now, if for whatever reason the dapp can't reach the arweave gateway, it will then deploy a new contract every time the page refreshes.